### PR TITLE
added user.ejs

### PR DESCRIPTION
**Pull request by JenniceLee0318**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
-const express = require('express')
-const path = require('path')
+const express = require('express');
+const path = require('path');
+var url = require('url');
+var fs = require("fs");
```
Comment (sanjayalwani) [inline index.js]: Where do you use url or fs? Can we remove this

File: index.js [hunk 2]
```diff
@@ -13,12 +16,28 @@ if(process.env.NODE_ENV!="production"){
...
+var bodyParser = require('body-parser')
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
 app.use(express.static(path.join(__dirname, 'public')));
 
+app.use(express.json());
+app.use(express.urlencoded({extended:false}));
 app.set('views', path.join(__dirname, 'views'));
+app.set('pages', path.join(__dirname, 'pages'));
```
Comment (sanjayalwani) [inline index.js]: what is pages for isn't everything in views?

File: index.js [hunk 3]
```diff
@@ -13,12 +16,28 @@ if(process.env.NODE_ENV!="production"){
 }
 
 const app = express();
-
+var bodyParser = require('body-parser')
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
```
Comment (sanjayalwani) [inline index.js]: why bodyparser urlencoded and express urlencoded one with extended as false and one as true

File: index.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
-const express = require('express')
-const path = require('path')
+const express = require('express');
+const path = require('path');
+var fs = require("fs");
```
Comment (sanjayalwani) [inline index.js]: where do you use this

File: index.js [hunk 5]
```diff
@@ -30,5 +47,44 @@ app.get('/feed', (req, res) => {
...
+//User Profile page
+// app.get('/user/:login',function(req,res,next){
+//   var login = req.params.login;
+//   var sql = "SELECT * FROM Users where login = $1";
+//   pool.query(sql,[login],function(err,data){
+//     if(err) console.error(err);
+//     res.render('pages/user',{title:"User Profile",row:data.rows[0]});
+//
+//   });
+// });
```
Comment (sanjayalwani) [inline index.js]: clean up comments

**Review by sanjayalwani — CHANGES_REQUESTED** [review]

File: package.json [hunk 6]
```diff
@@ -11,7 +11,7 @@
     "test": "node test.js"
   },
   "dependencies": {
-    "ejs": "^2.7.4",
+    "ejs": "^3.1.3",
     "express": "^4.17.1",
```
Comment (mann-c) [inline package.json]: delete this file from the pull request

File: views/pages/index.ejs [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
-  <% include ../partials/header.ejs %>
+  <%- include ('../partials/header.ejs') %>
 </head>
 
 <body>
 
-  <% include ../partials/nav.ejs %>
+  <%- include ('../partials/nav.ejs') %>
 
 <div class="jumbotron text-center">
```
Comment (mann-c) [inline views/pages/index.ejs]: delete this file from the merge request

**Review by mann-c — CHANGES_REQUESTED** [review]

**Review by mann-c — APPROVED** [review]

**Review by sanjayalwani — APPROVED** [review]

**Pull request merged**